.vscode/c_cpp_properties.json

```diff
@@ -0,0 +1,64 @@
+{
```

Comment:
@hugoferrari These are VS Code project configuration files. Is there any reason to commit these?